Tests/IntegrationTests/contentModule.js

```diff
@@ -3,9 +3,9 @@ import ReactSelector from 'testcafe-react-selectors';
 
 import Page from './pageModel';
 
-const section = (name, testFunc) => {
+const section = async (name, testFunc) => {
     console.log('\x1b[44m%s\x1b[0m', name);
-    testFunc();
+    await testFunc();
```

Comment:
How does running it async makes it any different in terms of false-positivity? If one of the async parts of code asserts an error, the whole test should die anyways, no?

Reply:
The test runner calls all `section`s and then terminates without executing any assertions, because it doesn't wait for the promises to resolve. This results in a zero exit code.

The test is positive, because it has no chance to fail before it terminates :)